app/services/ocean_service.rb

```diff
@@ -0,0 +1,62 @@
+class OceanService
+  ISLAND_SCATTER_RADIUS = 200
+  CENTER = [500,500]
+
+  def initialize(number_of_islands)
+    @number_of_islands = number_of_islands
+  end
+
+  def call
+    { islands: islands }
+  end
+
+  def islands
+    if @number_of_islands.zero? || @number_of_islands.nil?
+      []
+    else
+      size = island_size
+      biggest_island_size = size.sort.last
+      generate_positions(biggest_island_size).each_with_index.map { |position,i|IslandService.new(position, size[1-i]).call}
```

Comment:
please change to `do ... end` and add some spaces here. 
btw there is also a good rubocop-linter plugin for atom.